src/methods.jl

```diff
@@ -92,18 +105,20 @@ julia> subtract_overscan(frame, "[4:5, 1:1]", dims = 2)
 ```
 
 # See Also
-* [`subtract_overscan!`](@ref)
+[`subtract_overscan!`](@ref)
 """
-subtract_overscan(frame::AbstractArray, idxs; dims = axes_min_length(idxs)) = subtract_overscan!(deepcopy(frame), idxs, dims = dims)
-
+subtract_overscan(frame, idxs; kwargs...) = subtract_overscan!(deepcopy(frame), idxs; kwargs...)
+subtract_overscan(frame::ImageHDU, idxs; kwargs...) = subtract_overscan!(getdata(frame), idxs; kwargs...)
+subtract_overscan(frame::ImageHDU, key::Symbol; kwargs...) = subtract_overscan!(getdata(frame), read_header(frame)[string(key)]; kwargs...)
```

Comment:
it doesn't matter right now because the behavior is identical, but you could have also used this version, which dispatches to the function a line above this.  
```suggestion
subtract_overscan(frame::ImageHDU, key::Symbol; kwargs...) = subtract_overscan(frame, read_header(frame)[string(key)]; kwargs...)
```